### ISSUE TITLE
Light / Dark mode Toggle

### ISSUE THREAD

**Issue by LouisLouiseKay** [funmusicplace/mirlo#125]
It'd be more enjoyable to be able to toggle Light/Dark mode with a logo-button in the top right of the screen (or somewhere else easily accessible at all times).

**Comment by timglorioso** [comment]
I'll take this, after closing the color issue which I have not created yet :) will link

**Comment by LouisLouiseKay** [comment]
I personally change it all the time on websites (depending on fatigue, time of day, visual layout, my mood) and I much prefer proemonent toggles this is why I'm asking this especially on desktop where I don't want to have to change my default settings for each and every site I'm on. But if it's easily accessible enough in parameters I can live with it.

**Comment by timglorioso** [comment]
@LouisLouiseKay Does your desktop OS offer dark/light mode? If so, do you find it desirable for individual websites to deviate from that?

**Comment by LouisLouiseKay** [comment]
Yes I do that's what I meant above. My system settings are on dark theme by default but I want to be able to swap at any time on any website for a bunch of personal and shifting reasons. When I see this symbol in the top bar of a website I'm always relieved. 
![Screenshot_2023-10-15-20-53-29-296-edit_com android chrome](https://github.com/funmusicplace/mirlo/assets/77789607/b0a266c7-27af-4fe9-9ac4-bac5b3ed46e6)

**Comment by LouisLouiseKay** [comment]
Sounds fine for me :-)
